index.js

```diff
@@ -89,10 +90,9 @@ function Pickup (opts) {
 
     if (isSet) {
       // First wins, except 'content:encoded' summary of reasonable length…
-      if (key === 'summary' && (name !== 'content:encoded' || t.length > 4096)) {
+      if (key === 'summary' && !(name === 'content:encoded' && t.length < 4096)) {
         return
-      // …and pubDate.
-      } else if (key === 'updated' && name !== 'pubDate') {
+      } else if (Array.from(map.keys()).indexOf(name) > Array.from(map.keys()).indexOf(prev)) {
```

Comment:
We have to measure the performance of this.

Reply:
0.05 ms is what I’m seeing here, running the tests.

Reply:
Compared to 0.005 ms before. Initializing objects is expensive. Optimization required!

Reply:
Been able to shave it off to 0.01 ms.